Gemfile

```diff
@@ -41,15 +49,16 @@ end
 
 group :test do
   gem "poltergeist"
+  gem "capybara", ">= 2.6.2"
   gem "database_cleaner"
-  gem "shoulda-matchers", require: false
+  gem "shoulda-matchers"
   gem "simplecov", require: false
-  gem "sinatra"
+  gem "sinatra", github: "sinatra/sinatra"
```

Comment:
My suggestion was to add the comments for every line where it isn't simply doing `gem "thing"`.